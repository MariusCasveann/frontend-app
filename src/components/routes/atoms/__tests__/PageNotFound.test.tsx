import * as React from 'react';
import renderer from 'react-test-renderer';
// components
import PageNotFoundPresenter from '../PageNotFound';

describe('PageNotFoundPresenter', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<PageNotFoundPresenter />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
