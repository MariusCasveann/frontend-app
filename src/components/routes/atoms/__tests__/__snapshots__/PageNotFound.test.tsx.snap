// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFoundPresenter renders correctly 1`] = `
<div
  className="dashboard-wrapper"
>
  Page not found
</div>
`;
